src/components/HomeNewReleaseSection/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import NewReleaseBanner from '../NewReleaseBanner'
+import NewRelease from '../NewRelease'
+
+const HomeNewReleaseSection = () => {
+  return (
+    <section className="home-section home-section--light">
+      <NewReleaseBanner />
+      <NewRelease leftContent={true} />
+      <NewRelease />
```

Comment:
I have two concerns
1. Should we rename it to `NewReleaseItem` ?
2. Please create `enum` and pass it as `props` of this component.  (See example in `Footer` component)

PS. Naming is up to you, if you don't like the name `imagePosition` and `Position`, feel free to suggest. 

```suggestion
      <NewReleaseItem imagePosition={Position.right} />
      <NewReleaseItem imagePosition={Position.left} />
```